### PR TITLE
Finish off the labels

### PR DESCRIPTION
**Pull request by RDIL**
now my hands hurt

### CODE REVIEW

**Comment by RDIL** [comment]
@PoshChan please retry CI

**Comment by PoshChan** [comment]
@RDIL, successfully started retry of `docker-PR`

**Comment by RDIL** [comment]
@kiazhi I think I broke parrot again 🤕

File: release/lts/alpine39/test-deps/docker/Dockerfile [hunk 1]
```diff
@@ -33,27 +33,7 @@ RUN apk add --no-cache --virtual .pipeline-deps readline linux-pam \
...
-      org.label-schema.vendor="PowerShell" \
-      org.label-schema.version=${PS_VERSION} \
-      org.label-schema.schema-version="1.0" \
-      org.label-schema.vcs-ref=${VCS_REF} \
-      org.label-schema.docker.cmd="docker run ${IMAGE_NAME} pwsh -c '$psversiontable'" \
-      org.label-schema.docker.cmd.devel="docker run ${IMAGE_NAME}" \
-      org.label-schema.docker.cmd.test="docker run ${IMAGE_NAME} pwsh -c Invoke-Pester" \
-      org.label-schema.docker.cmd.help="docker run ${IMAGE_NAME} pwsh -c Get-Help" \
-      com.azure.dev.pipelines.agent.handler.node.path="/usr/local/bin/node"
+LABEL com.azure.dev.pipelines.agent.handler.node.path="/usr/local/bin/node"
```
Comment (TravisEz13) [inline release/lts/alpine39/test-deps/docker/Dockerfile:36]: thanks for recognizing this needs to stay

**Comment by TravisEz13** [comment]
Thanks again @RDIL.  Conference Q&A today:   https://powershell.love

**Comment by RDIL** [comment]
@TravisEz13 the site refuses to connect for me

**Comment by TravisEz13** [comment]
Direct link: http://powershell.one/psconfeu/psconf.eu-2020/about

**Comment by kiazhi** [comment]
> @kiazhi I think I broke parrot again 🤕

@RDIL I don't think you have broke the parrotsec. It seems to be an intermittent network connectivity issue when fetching packages from their repo that is causing the issue. I can't replicate the CI error locally on my macOS. Anyway, I'm going to change something on parrotsec dockerfile. No worry. 😁

**Pull request merged**